modules/nf-core/merquryfk/merquryfk/main.nf

```diff
@@ -55,4 +67,20 @@ process MERQURYFK_MERQURYFK {
         r: \$( R --version | sed '1!d; s/.*version //; s/ .*//' )
     END_VERSIONS
     """
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+    def FASTK_VERSION = 'f18a4e6d2207539f7b84461daebc54530a9559b0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def MERQURY_VERSION = '8ae344092df5dcaf83cfb7f90f662597a9b1fc61' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    touch ${prefix}.completeness.stats
+    touch ${prefix}.qv
+    touch ${prefix}._.qv
```

Comment:
Why is this name `._.`?

Reply:
assembly qv file is another qv file that needs to be in the output, its pattern is usually ${prefix}.*.qv, what do you suggest to change to please? @SPPearce . thanks

Reply:
Ok, if that is the sort of name that gets generated out of the tool that is fine.